mqtt/mqtt-broker/src/settings/mod.rs

```diff
@@ -160,21 +160,21 @@ impl Default for RetainedMessagesConfig {
 
 #[derive(Debug, Clone, PartialEq, Deserialize)]
 pub struct SessionPersistenceConfig {
-    path: PathBuf,
+    folder_path: PathBuf,
```

Comment:
curious why not just `path`? it belongs to the persistence config section thus describes the path to persisted data. `file_path` is bad, I agree

Reply:
I just wanted to emphasize that his is a folder path, not a file path. IMO it is unclear from just "path".